### ISSUE TITLE
Any plan to support fastvideo distill training?

### ISSUE THREAD

**Issue by trouble-maker007** [a-r-r-o-w/finetrainers#127]
Any plan to support [FastVideo](https://github.com/hao-ai-lab/FastVideo) in diffusers?

**Comment by a-r-r-o-w** [comment]
Yes, it would be awesome to support the fastvideo distilled models!

@jzhang38 @foreverpiano Would you be okay with hosting the Diffusers-format converted weights? Would be happy to help with anything since we also briefly interacted in some of the other PRs :) Glad to see your parallelization training project worked so greatly!

We have also discussed briefly to eventually support a parallization backend natively in Diffusers but nothing concrete and it will take time to design and make available for all models without completely fusing it into every implementation.

**Comment by sayakpaul** [comment]
Would be awesome to support distillation!

**Comment by jzhang38** [comment]
@a-r-r-o-w Yes! I will work on converting the weights/

**Comment by a-r-r-o-w** [comment]
Thank you @jzhang38 🤗 Looking forward to it! Also thank you for releasing the datasets used for distillation ~ it will be very helpful for experimentation
